.gitignore

```diff
@@ -10,9 +9,15 @@ GNUmakefile
 Makefile
 *.lst
 
+temp/
+tmp/
```

Comment:
While in theory these are somewhat "generic", with only a couple of files added to Phobos each year I doubt that we ever run into the problem that we want to create a `tmp` module.